velox/common/memory/MappedMemory.h

```diff
@@ -55,6 +55,8 @@ class MappedMemory {
   static constexpr uint64_t kPageSize = 4096;
   static constexpr int32_t kMaxSizeClasses = 12;
   static constexpr int32_t kNoOwner = -1;
+  // Marks allocation via allocateBytes, e.g. StlMappedMemoryAllocator.
+  static constexpr int32_t kMallocOwner = -14;
```

Comment:
Should we have an owner management class in case multiple owners appear for the same usecase